test/e2e/config/e2e_conf.yaml

```diff
@@ -18,7 +18,7 @@ providers:
 - name: cluster-api
   type: CoreProvider
   versions:
-  - name: v0.4.8
+  - name: ${CAPI_FROM_RELEASE} #v0.4.8
```

Comment:
It is better not to specify the version (i.e #v0.4.8) even as a comment here and in all other cases in this file since they will get outdated quickly and each time needs to be changed when CAPI makes a new release.